### ISSUE TITLE
Can't select backward with space+h

### ISSUE THREAD

**Issue by patrici-o** [jasonrudolph/keyboard#48]
**tl;dr: I can't select backwards in (S)uper (D)uper mode by holding down <kbd>space</kbd> and tapping <kbd>h</kbd>.**

Further info:

- All other directions (<kbd>j</kbd> / <kbd>k</kbd> / <kbd>l</kbd>) + <kbd>space</kbd> *do* work.

- If I modify the line in ```super.lua``` from
```charactersToModifers[' '] = 'shift'```
to
```charactersToModifers['v'] = 'shift'```, then backward selection *does* work. I could leave it like that, but the cool thing about <kbd>space</kbd> is the thumb accessibility.

- If I'm inside (S)uper (D)uper mode and I use <kbd>shift</kbd> instead of <kbd>space</kbd> for selection, I get mixed results:
    - If I start by pressing <kbd>shift</kbd>, it gets "captured" as a modifier to either <kbd>s</kbd> or <kbd>d</kbd> and I get the macOS tooltip for selecting a special character.
    - If I first tap <kbd>h</kbd> at least once, and *then* start to hold down <kbd>shift</kbd>, selection works properly.

- macOS 10.13.6 (High Sierra), Hammerspoon 0.9.73, Karabiner-Elements 12.1.0

Many thanks for your help!

**Comment by patrici-o** [comment]
Ah, just saw #24. I wanted to add that I *can't* select backwards with the built-in keyboard on my MacBook Pro, but I *can* select backwards with an external USB keyboard.
I used [Ukelele](https://scripts.sil.org/ukelele) to check which keys were registering--the <kbd>h</kbd> doesn't register.

**Comment by jasonrudolph** [comment]
:wave: Hi @patrici-o: Thanks for the superb write-up in this issue. 👏

And kudos for digging into this issue and figuring out that it's a hardware problem:

> I *can't* select backwards with the built-in keyboard on my MacBook Pro, but I *can* select backwards with an external USB keyboard.

Since this behavior is the result of a hardware limitation on the built-in MacBook Pro keyboard, I'm going to close this issue.
